shader/coarse.wgsl

```diff
@@ -170,6 +176,9 @@ fn main(
     var render_blend_depth = 0u;
     var max_blend_depth = 0u;
 
+    let blend_offset = cmd_offset;
+    cmd_offset += 1u;
```

Comment:
BTW we can use `++` in WGSL. I'm just used to not having it in Rust.

Reply:
Me too and I've come to prefer it.